src/components/LoanCollections/KivaClassicLoanCarousel.vue

```diff
@@ -226,8 +266,18 @@ export default {
 		},
 		// TODO: Review all tracking cateogries
 		onInteractCarousel(interaction) {
-			this.$kvTrackEvent('carousel', 'click-carousel-horizontal-scroll', interaction);
+			if (this.newHomeExp) {
+				this.$kvTrackEvent('swipe-carousel', 'swipe-carousel-horizontal-scroll', interaction);
+			} else {
+				this.$kvTrackEvent('carousel', 'click-carousel-horizontal-scroll', interaction);
```

Comment:
These tracking changes seem really arbitrary to make a switch for an exp. We should just let them know there is already tracking on interactions with the carousel. I'm not sure if it's possible but you might be able to use the value returned from `interaction` to better understand the type of interaction.